modules/amf-registration/amf-registration-web/src/main/java/com/liferay/amf/web/actions/RegisterAccountMVCActionCommand.java

```diff
@@ -41,12 +42,13 @@ protected void doProcessAction(ActionRequest actionRequest, ActionResponse actio
         String securitysecurityAnswer = ParamUtil.getString(actionRequest, "securityAnswer");
         String acceptedTou = ParamUtil.getString(actionRequest, "acceptedTou");
 
-
-
-        _accountLocalService.createAccount(accountId,firstName,lastName,emailAddress,accountName,gender,birthday,password,
+//        //Call the service to add a new Account
+        _accountLocalService.createAccount(firstName,lastName,emailAddress,accountName,gender,birthday,password,
                 homePhone,mobilePhone,address,address2,city,statezip,securityQuestion,securitysecurityAnswer,
                 acceptedTou);
-        System.out.println(("Account created sucefully"));
+
+      System.out.println("Account created sucefully");
```

Comment:
fix typo and please remove any logs in console. if u want to lunch any logs, you can use something like this instead:

```
_log.info("Account created successfully");

[...]

private static final Log _log = LogFactoryUtil.get(NameOfTheClass.class.getName());
```